events/interfaces.go

```diff
@@ -26,6 +26,9 @@ type Helper interface {
 	// An action must not spawn itself to avoid an infinite loop.
 	SpawnAs(name string, action string, args ...string) error
 
+	// Spawned returns true if the action is running in a child process.
+	Spawned() bool
```

Comment:
Should it return the pid of the child process in the same way `unshare()` and `fork()` work?

Reply:
I would avoid that implementers rely on that, mainly because the underlying mechanism does not behave exactly as `unshare()` and `fork()` and may change in the future.

Also, `SpawnAs()` waits for the child to complete, so there is no need to wait for a pid.
I have rewritten the function comments to make it more clear.